src/app/note/terra-note.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="note"
```

Comment:
Wäre besser, wenn du hier die Info-Box nutzen würdest und nur die relevanten styles in der .scss überschreibst!